resources/views/calendars/events/_subform.blade.php

```diff
@@ -82,3 +82,10 @@
         <p class="help-block">{!! __('entities/events.helpers.characters', ['more' => link_to_route('helpers.age', __('crud.actions.find_out_more'), null, ['target' => '_blank'])]) !!}</p>
     </div>
 @endif
+@if (!empty($entity) && $entity->typeId() == config('entities.ids.location') || $entity->typeId() == config('entities.ids.family') || $entity->typeId() == config('entities.ids.organisation'))
```

Comment:
`$entity->typeId() == config('entities.ids.location') || $entity->typeId() == config('entities.ids.family') || $entity->typeId() == config('entities.ids.organisation')` can be replaced with an `in_array($entity->typeId(), [...])`